pyscriptjs/src/utils.ts

```diff
@@ -109,3 +111,35 @@ export function getAttribute(el: Element, attr: string): string | null {
     }
     return null;
 }
+
+export function calculatePaths(fetch_cfg: FetchConfig[]) {
+    const fetchPaths: string[] = [];
+    const paths: string[] = [];
+    fetch_cfg.forEach(function (each_fetch_cfg: FetchConfig) {
```

Comment:
[question] just my curiosity but: what is the advantage of using `forEach` instead of a normal `for (each_fetch_cfg of FetchConfig)`?

Reply:
I don't know to be honest except that it is considered a standard practice I guess? If not, I can change. Let me know.

Reply:
I don't really know because I'm not an expert of JS/TS.
In all the rest of the code we use `for(... of ...)`, that's why I wondered whether here it was different